packages/tools/devtools/devtools-core/src/data-visualization/DefaultVisualizers.ts

```diff
@@ -15,11 +15,11 @@ import { SharedMatrix } from "@fluidframework/matrix";
 import { SharedString } from "@fluidframework/sequence";
 import { type ISharedObject } from "@fluidframework/shared-object-base";
 import type { ISharedTree } from "@fluidframework/tree/internal";
-import { SharedTree, encodeTreeSchema } from "@fluidframework/tree/internal";
-
```

Comment:
You'll want to restore this blank line - it was added to prep for automated import sorting, which is in the works.